dts/bindings/pinctrl/nuvoton,npcm-pinctrl.yaml

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) 2024 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The Nuvoton pin controller is a singleton node responsible for controlling
+  pin function selection and pin properties. For example, you can use these
+  nodes to select peripheral pin functions.
+
+  Here is a list of supported standard pin properties:
+    - bias-pull-down: Enable pull-down resistor.
+    - bias-pull-up: Enable pull-up resistor.
+
+  An example for the NPCM4 family, include the chip-level pinctrl DTSI file in the
+  board-level DTS:
+
+    #include <nuvoton/npcm/npcm4/npcm4-pincfg-default.dtsi>
+
+  We want to use the I2C0_0 port of the NPCM400F controller and enable the
+  internal 3.3V pull-up if its I2C frequency won't exceed 400kHz.
+
+  To change a pin's pinctrl default properties, add a reference to the pin in
+  the board's DTS file and set the properties in the child node of pinctrl as below:
+
+    i2c1a_scl_sda_gpio60_61: periph-i2c1a {
+      scl {
+        pinmux = <GPIO60__RSTOUT2__SCL1A 2>;  /* Set the pinmux to SCL1A */
+        lvol = <0>;                           /* Set the voltage level to 3.3V */
```

Comment:
this looks unreadable, description should be clear from a DT point of view: standard `power-source` may be used, or, `vendor,voltage-level-mv = <3300>`.

Reply:
Fixed. Also rename the `pupd-*`, `lv-*` and `dev-ctl` properties to make it clear in DT.